app/src/main/java/com/example/gravit/login/LoginActivity.kt

```
@@ -88,146 +83,122 @@ fun LoginScreen (
+            ) {
+                SocialLoginButton(
+                    text = "Google로 시작하기",
+                    backgroundColor = AppColor.bg1,
+                    contentColor = AppColor.CTA_secondary_text,
+                    logoResId = R.drawable.google_logo,
+                    onClick = {
+                        scope.launch {
+                            try {
+                                val idToken = loginWithGoogle(context)
+                                viewModel.sendIdTokenToServer("google", idToken)
+                            } catch (e: NoCredentialException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)